src/index.ts

```diff
@@ -88,15 +88,6 @@ const migrationInit = async function () {
   // Create an authentication helper instance using an API key
   const authHelper = await withAPIKey(apiKey);
 
-  // TODO: We still create a V1 client for now while we are in the process of converting all APIs to V2
-  const apiKeyV1 = apiKey;
-  const authHelperV1 = await withAPIKey(apiKeyV1);
-  const clientV1 = new LocationClient({
-    region: "us-west-2", // Region containing Amazon Location resource
-    customUserAgent: `migration-sdk-${PACKAGE_VERSION}`, // Append tag with SDK version to the default user agent
-    ...authHelperV1.getClientConfig(), // Configures the client to use API keys when making supported requests
-  });
-
```

Comment:
It wouldn't let me comment above, but we should remove the `LocationClient` import above since it is no longer used